prettyprinter-configurable/Setup.hs

```diff
@@ -7,18 +7,20 @@ module Main where
 #define MIN_VERSION_cabal_doctest(x,y,z) 0
 #endif
 
-#if MIN_VERSION_cabal_doctest(1,0,0)
+-- stylish-haskell is unable to parse import lines that are interspersed with
+-- other code, hence why we resort to using two #ifs instead of one.
```

Comment:
As per the comment, this workaround is necessary to please stylish-haskell

Reply:
Nice workaround!